Software/Python/easygopigo3.py

```diff
@@ -476,6 +476,21 @@ def reset_encoders(self):
         self.offset_motor_encoder(self.MOTOR_LEFT,self.get_motor_encoder(self.MOTOR_LEFT))
         self.offset_motor_encoder(self.MOTOR_RIGHT,self.get_motor_encoder(self.MOTOR_RIGHT))
 
+    def read_encoders(self):
```

Comment:
Okay, we'll then go with `read_motor_position()` instead of `read_encoders`.The `GoPiGo3` is just a bit more complicated/uncomfortable when you need to determine the speed or when required to read the encoders all the time - it happened to me.